andela/Lit-Kigali1-Frontend
# #163519097 Implement signup page using React

#### What does this PR do?
Implement signup page using React

#### Description of Task to be completed?
- Add signUp component
- write feature tests

#### How should this be manually tested?
- Clone the repository
- Run `npm run start`
- Navitage to `/signup` url using any browser
- Fill and submit the form
- Check your inbox for the email confirmation link
- Click on the email confirmation link

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
`#163519097`

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/29614532/56124247-81349f80-5f76-11e9-864f-add85f57df28.png)


![image](https://user-images.githubusercontent.com/29614532/56124257-88f44400-5f76-11e9-873f-b880472338c9.png)



![image](https://user-images.githubusercontent.com/29614532/55786280-3fa67f00-5ab4-11e9-8cbf-cba1cab7ffdb.png)


#### Questions:
N/A
